addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -360,14 +360,24 @@
         <t t-set="go_back_url" t-valuef="/forum/#{ slug(forum) }/"/>
     </t>
     <t t-else="" t-set="_no_results_title">Oops!</t>
+    <t t-if="filters == 'unanswered'" t-set="filtered_results">
+        Sorry, we could not find any <b>unanswered</b> results
+    </t>
+    <t t-elif="filters == 'solved'" t-set="filtered_results">
+        Sorry, we could not find any <b>solved</b> results
+    </t>
+    <t t-if="filters == 'unsolved'" t-set="filtered_results">
+        Sorry, we could not find any <b>unsolved</b> results
+    </t>
     <div t-attf-class="#{ 'o_caught_up_alert' if queue_type else '' } row g-0 justify-content-center #{ 'd-none' if hide_alert else '' }">
         <div class="p-3 d-flex flex-column align-items-center">
             <img t-if="queue_type" src="/website_forum/static/src/img/tasks.svg" class="img img-fluid mb-3" width="200" alt="Animation of a pen checking a checkbox"/>
             <img t-else="" src="/website_forum/static/src/img/empty.svg" class="img img-fluid mb-3" width="150" alt="Empty box"/>
             <h5 t-out="_no_results_title"/>
             <span t-if="result_msg" t-out="result_msg"/>
             <span t-else="">
-                Sorry, we could not find any <b t-if="filters in ['unanswered', 'solved', 'unsolved']" t-out="filters"/> results
+                <t t-if="filtered_results" t-out="filtered_results"/>
```

Comment:
I think it will work in debug mode, but not in non debug mode
(Qweb do `values.get('filtered_results')` in debug but do `values['filtered_results']` in non debug if I'm not wrong)

so we should add `<t t-set="filtered_results" t-value="False"/>` before the others

Reply:
Maybe this changed at some point (in 9ce5bc88?) but it looks like it's working like this. I can still add is as an `else` if it is cleaner though.
(The rest is done, thanks for the validation :))

Reply:
I did, thanks! 
(email didn't put it here)